interfaces/product.ts

```diff
@@ -7,8 +7,8 @@ export interface APIProductI {
 }
 
 export interface ProductI {
-  image: String,
-  link: String,
-  text: String,
-  title: String
+  image: string
+  link: string
+  text: string
+  title: string
```

Comment:
`String` is an object while `string` is a primitive (literal).

https://www.geeksforgeeks.org/what-is-the-difference-between-string-and-string-in-typescript/
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String